src/app/components/FeedFilter/FeedFilter.tsx

```diff
@@ -0,0 +1,28 @@
+import FeedFilterElement from '../FeedFilterElement/FeedFilterElement';
+import classes from './FeedFilter.module.css';
+
+type FeedFilterProps = {
+  selectedFilter?: string;
+  onFilterChange: (filterName: string) => void;
```

Comment:
If this is one state it would be cool to name it the same way, like: `[selectedFilter, setSelectFilter]`

Then is the onClick also more understandable: -> `onClick(() => setSelectedFilter())`